frog/.meteor/packages

```diff
@@ -20,7 +20,7 @@ shell-server@0.3.1            # Server-side component of the `meteor shell` comm
 
 insecure@1.0.7                # Allow all DB writes from clients (for prototyping)
 accounts-ui@1.3.0
-accounts-password@1.5.1
+accounts-password
```

Comment:
Nit: specify which version we are using to avoid version related issues in the future

Reply:
I ran meteor-add and for some reason it removed the version..

Reply:
Weird... I'll leave it up to you to fix it if you think it is important